src/pkg/sif/sif.go

```diff
@@ -44,9 +43,10 @@ func FromSandbox(sandbox *image.Sandbox, imagePath string) (*SIF, error) {
 
 	fmt.Println(string(mksfsout))
 
-	sif := buildcfg.SBINDIR + "/sif"
-
-	sifCmd := exec.Command(sif, "create", "-P", squashfsPath, "-f", "SQUASHFS", "-p", "SYSTEM", "-c", "LINUX", imagePath)
+	sif := []string{"sif", "create", "-P",
```

Comment:
It's confusing to call this `sif`, since it's no longer the location of the SIF binary. What about `args`?